Eastern-Research-Group/csb-rebate-forms-app
# Feature/update default rebate year

## Related Issues:
* CSBAPP-422

## Main Changes:
Update the app to set the initially selected rebate year via an environment variable (used in the config API endpoint response, which is called when the user initially logs into the app).

## Steps To Test:
1. Navigate to the dashboard. Ensure the rebate year is currently set for "2023" (as that's the current value used in the env variable for dev and staging).
2. Change the rebate year selection and navigate to another page. Navigate back to the dashboard (or helpdesk page) and ensure the rebate year remains what you selected.
3. If running the app locally, try changing the rebate year via the env file and ensure the value you change (2022, 2023, or 2024) is reflected as the initial rebate year selected in the app.
